src/components/Routing.js

```diff
@@ -123,6 +125,13 @@ class Routing extends Component {
 
             handleClearAlert()
         })
+
+        let id = Mixpanel.get_distinct_id()
+        Mixpanel.identify(id)
+        Mixpanel.people.set({enabled_2FA: this.props.account.twoFactor, can_enable_two_factor: this.props.account.canEnableTwoFactor})
+        if(this.props.account.accountId){
```

Comment:
nit: please format like the rest of the code, e.g. here should be `if (this.props.account.accountId) {`. Same for other similar instances, not just in this line.